package-lock.json

```diff
@@ -4870,7 +4870,7 @@
     },
     "minimist": {
       "version": "0.0.8",
-      "resolved": "http://registry.npmjs.org/minimist/-/minimist-0.0.8.tgz",
+      "resolved": "https://registry.npmjs.org/minimist/-/minimist-0.0.8.tgz",
```

Comment:
@johnsaigle @driusan is this file supposed to be committed in this PR?

Reply:
Not unless the PR updates a nodejs dependency. So I don't think so.